### PR TITLE
Update link, navbar and Dashboard page, add spinner

### LINKED ISSUE #44: Add spinner
There should be a spinner appears while the page is loading to avoid blank screen.

### LINKED ISSUE #45: Fix problems with width of Dashboard table
The width of the screen should be dynamic instead of fixed as it's right now :'( I'll fix it soon :'(

### LINKED ISSUE #46: Replace Search Box in Dashboard
Replace Search Box in Dashboard with the newly created one which can search many fields instead of just phone number

### LINKED ISSUE #49: Update navigation bar
Update navigation bar with the new link

### PR DESCRIPTION
**Pull request by chalalala**
- Update link in website
- Update modal in Dashboard
- Fix bug in width of Dashboard
- Update links and fix bug in Navigation Bar
- Add spinner for pages

### CODE REVIEW

**Review by PhuongPhg — COMMENTED** [review]
image
- The nav bar in dashboard has been fixed, but I wonder can we remove the blank space? 
- When I click log out, i think it's better to navigate to homepage
- When i click on 'Already have account', it navigates to the homepage, I think it's should be change to signup.

The rest is cool now 👍

**Comment by chalalala** [comment]
@PhuongPhg I've made some changes. Could you please check it?

File: src/views/dashboard.js [hunk 1]
```diff
@@ -377,12 +377,11 @@ export default function Dashboard() {
 
 const useStyles = makeStyles((theme) => ({
 	container: {
-		width: "max-content",
+		maxWidth: "max-content",
 		minWidth: "100vw",
```
Comment (PhuongPhg) [inline src/views/dashboard.js]: It's still has some space between the table and the right. I think change `minWidth` to be less then `100vw` can fixed the problem.

**Review by PhuongPhg — APPROVED** [review]
It still have some space in the right of table. But it's not a big problem.

The rest is fine now. You can merge it.

**Comment by chalalala** [comment]
@PhuongPhg I've made some changes, again :'( Could you help me check it :'( ?

**Review by PhuongPhg — APPROVED** [review]
Everything is okay now. Let's merge it

**Pull request merged**